lib/bootstrap/bundler.rb

```diff
@@ -46,8 +46,9 @@ def setup!(options = {})
       # make sure we use our own installed bundler
       LogStash::Rubygems.patch!
       ::Gem.clear_paths
-      ::Gem.paths = ENV['GEM_HOME'] = ENV['GEM_PATH'] = Environment.logstash_gem_home
 
+      ENV['GEM_HOME'] = ENV['GEM_PATH'] = LogStash::Environment.logstash_gem_paths
+      ::Gem.paths = ENV
```

Comment:
Gem.paths takes ENV now? That seems not right. ENV is a Hash thing, but I think Gem.paths= takes an array?